src/components/Popup/Popup.js

```diff
@@ -244,6 +248,7 @@ class Popup extends PureComponent {
       >
         <div
           role="presentation"
+          key={key}
           ref={popupElement => {
             this.setState({ popupElement });
```

Comment:
doing this is on non-sense, use createRef instead

Reply:
The problem is that I need to measure the dimensions of the popup _after_ the popup was rendered. I use the key for forcing a re-render if the coordinate or the feature was updated. Consequently the state is updated and I trigger another component update.

Maybe there's a better solution but I didn't find one so far.